rpc/wrtc_signaling_answerer.go

```diff
@@ -212,9 +213,21 @@ func (ans *webrtcSignalingAnswerer) startAnswerer() {
 				continue
 			}
 
-			// `client.Recv` waits, typically for a long time, for a caller to show up. Which is
-			// when the signaling server will send a response saying someone wants to connect.
-			incomingCallerReq, err := client.Recv()
+			var incomingCallerReq *webrtcpb.AnswerRequest
+			for {
+				// `client.Recv` waits, typically for a long time, for a caller to show up. Which is
+				// when the signaling server will send a response saying someone wants to connect. It
+				// can also receive heartbeats every 15s. Discard heartbeats.
```

Comment:
What's the useful side-effect of having the signaling server send heartbeats if we're just going to discard them?

I'm guessing the act of the signaling server calling `SendMsg(heartbeat)` is what can now trigger an error that the server handles by cleaning up server-side resources? Or maybe we expect the server's internal gRPC logic handles that error in a way that makes our problem go away?

Whatever it is, can we document that? I think someone seeing this will see these heartbeats are intentional, but not know how to measure if it's having the intended effect. I think it's fine if we don't exactly know the magic here, but leaving as much detail as we do know can be useful.

I can see a developer coming across this and wanting to send a "heartbeat" in response, as that's the usual pattern. Documentation can help that developer understand if there would be an observable impact of such a change.

Reply:
> I'm guessing the act of the signaling server calling SendMsg(heartbeat) is what can now trigger an error that the server handles by cleaning up server-side resources?

This is my understanding: it's the error from heartbeating server-side that's really helping out here. I'll document a bit better.